cmd/k8s-operator/operator.go

```diff
@@ -646,6 +669,27 @@ func proxyClassHandlerForConnector(cl client.Client, logger *zap.SugaredLogger)
 	}
 }
 
+// proxyClassHandlerForConnector returns a handler that, for a given ProxyClass,
+// returns a list of reconcile requests for all Connectors that have
+// .spec.proxyClass set.
+func proxyClassHandlerForProxyGroup(cl client.Client, logger *zap.SugaredLogger) handler.MapFunc {
```

Comment:
We will need also need to reconcile any ProxyGroups to whom a default ProxyClass applies when that ProxyClass changes. We can make that change in a separate PR though- I think some other reconcilers might have the same issue.
Perhaps you could add a TODO here

Reply:
I think this one is also covered by the offline discussion to defer using the default?

Reply:
Yes